scripts/package-emulator.sh

```diff
@@ -1,33 +1,24 @@
 #!/bin/bash
-set -
-cd ..
-. setup.sh
+source build/envsetup.sh
+HOST_ARCH=$(get_build_var HOST_ARCH)
+lunch "aosp_${HOST_ARCH}-userdebug"
 PRODUCT_OUT=$(get_build_var PRODUCT_OUT)
 HOST_OUT=$(get_build_var HOST_OUT)
 OUT_DIR=$(get_abs_build_var OUT_DIR)
 
 EMULATOR_FILES=(\
-	.config \
-	load-config.sh \
-	run-emulator.sh \
-	${HOST_OUT}/bin/adb \
-	${HOST_OUT}/bin/emulator \
-	${HOST_OUT}/bin/emulator-arm \
-	${HOST_OUT}/bin/mksdcard \
-	${HOST_OUT}/bin/qemu-android-x86 \
-	${HOST_OUT}/lib \
-	${HOST_OUT}/usr \
-	development/tools/emulator/skins \
-	prebuilts/qemu-kernel/arm/kernel-qemu-armv7 \
-	${PRODUCT_OUT}/system/build.prop \
-	${PRODUCT_OUT}/system.img \
-	${PRODUCT_OUT}/userdata.img \
-	${PRODUCT_OUT}/ramdisk.img)
+       ${PRODUCT_OUT}/cache.img \
+       ${PRODUCT_OUT}/config.ini \
+       ${PRODUCT_OUT}/kernel-ranchu \
+       ${PRODUCT_OUT}/ramdisk.img \
+       ${PRODUCT_OUT}/system/build.prop \
+       ${PRODUCT_OUT}/system.img \
+       ${PRODUCT_OUT}/userdata.img \
+       ${PRODUCT_OUT}/userdata-qemu.img)
```

Comment:
I think we will need vendor.img and vendor-qemu.img as well.

Reply:
Thanks, I'll add them.